.github/workflows/spelling.yml

```diff
@@ -12,11 +12,40 @@ on:
       - "*-stable"
 
 jobs:
-  build:
+  spelling:
     name: Spell Check
-    runs-on: "ubuntu-latest"
+    permissions:
+      contents: read
+      pull-requests: read
+      actions: read
+    outputs:
+      followup: ${{ steps.spelling.outputs.followup }}
+    runs-on: ubuntu-latest
+    if: "contains(github.event_name, 'pull_request') || github.event_name == 'push'"
+    concurrency:
+      group: spelling-${{ github.event.pull_request.number || github.ref }}
+      # note: If you use only_check_changed_files, you do not want cancel-in-progress
+      cancel-in-progress: true
     steps:
-    - name: Checkout Repository
-      uses: actions/checkout@v3
-    - name: Check Spellings
-      uses: check-spelling/check-spelling@v0.0.19
+    - name: check-spelling
+      id: spelling
+      uses: check-spelling/check-spelling@v0.0.20
+      with:
+        suppress_push_for_open_pull_request: 1
+        checkout: true
+        post_comment: 0
+
+  comment:
+    name: Report
+    runs-on: ubuntu-latest
+    needs: spelling
+    permissions:
+      contents: write
+      pull-requests: write
```

Comment:
Can we do away with PR comments? I think inline comments (check-run annotations) are more than sufficient.

Reply:
Yes. 10 GitHub Annotations is sufficient. You may disable posting PR comments.